File: 06 (Summation)/p06.py
def solve():
    # Your solution code goes here
    pass

if __name__ == "__main__":
    tc = int(input())

    for _ in range(tc):
        number = input()
        first, last = int(number[0]), int(number[-1])

        # Now print the sum of the integer
        print(f"Sum = {first + last}")

File: 06 (Summation)/p06.java
import java.util.Scanner;

public class Main {
    static void solve() {
        // Your solution code goes here
    }

    public static void main(String[] args) {
        // #ifndef ONLINE_JUDGE
        //     // freopen("input.txt", "r", System.in);
        //     // freopen("output.txt", "w", System.out);
        // #endif

        Scanner scanner = new Scanner(System.in);

        int tc = scanner.nextInt();

        for (int i = 0; i < tc; i++) {
            String number = scanner.next();
            int first = Character.getNumericValue(number.charAt(0));
            int last = Character.getNumericValue(number.charAt(number.length() - 1));

            // Now print the sum of the integer
            System.out.println("Sum = " + (first + last));
        }
    }
}
